ext/opcache/jit/zend_jit.c

```diff
@@ -3086,7 +3091,11 @@ static int ZEND_FASTCALL zend_runtime_jit(void)
 	}
 
 	/* JIT-ed code is going to be called by VM */
-	return 0;
+#if GCC_GLOBAL_REGS
+	return; // ZEND_VM_CONTINUE
+#else
+	return op_array->opcodes; // ZEND_VM_CONTINUE
```

Comment:
We may skip leading RECV instructions. Actually, you should return the ``opline`` passed as argument or ``EX(opline)``.